geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/tools/AnalyticFeature.java

```diff
@@ -37,7 +37,10 @@ public static SimpleFeature createGeometryFeature(
 			final int zoomLevel,
 			final int iteration,
 			final long count ) {
-		assert (extraDimensionNames.length == extraDimensions.length);
+		if (extraDimensionNames.length != extraDimensions.length) {
```

Comment:
That's fine. I wanted something to go 'boom' here if the invariant came up not true.